src/app/TradeApp/index.tsx

```diff
@@ -21,11 +28,26 @@ const Trading = React.lazy(
     ),
 )
 
-const NavTools: React.FC = () => (
-  <Frame>
-    <footer>Tools</footer>
-  </Frame>
-)
+const PortfolioLink = styled.li`
```

Comment:
can you please move all styled component stuff into a `ComponentName.styled.ts` file?

Reply:
hehe, I'm happy to do it, but let's have a chat about it on the reasons why do it or why don't
I added a task also in the TODO